src/Model/LroPagedResponseGo.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using AutoRest.Core.Model;
+
+namespace AutoRest.Go.Model
+{
+    /// <summary>
+    /// Represents a response from a long-running operation that's also a paged operation.
+    /// This is a bit of a hack as the "list all" method isn't part of the code model and doing
+    /// it this way was the path of least resistance (we should fix this eventually).
```

Comment:
>we should fix this eventually). [](start = 54, length = 31)

maybe add an issue for this  so we don't loose track of it ?

Reply:
Opened https://github.com/Azure/autorest/issues/2752.